src/Particular.LicensingComponent.Contracts/ThroughputSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Particular.LicensingComponent.Contracts;
+
+using ServiceControl.Configuration;
+
+public class ThroughputSettings(string serviceControlQueue, string errorQueue, string transportType, string customerName, string serviceControlVersion)
+{
+    public static readonly SettingsRootNamespace SettingsNamespace = new("LicensingComponent");
+
+    public const string DatabaseNameKey = "RavenDB/ThroughputDatabaseName";
+    public const string DefaultDatabaseName = "throughput";
```

Comment:
Nit picking, but can this settings be specified by the user?
If not, then should we remove the work "Default"?

Reply:
It can be overridden in the settings.  The code for that is in SC Raven persistence project.